tds/src/main/java/thredds/featurecollection/cache/GridInventoryCacheChronicle.java

```diff
@@ -34,7 +34,7 @@ public class GridInventoryCacheChronicle implements InventoryCacheProvider {
    * @param cacheDir Path to the cache directory. This location will be created if it does not exist.
    * @throws IOException
    */
-  public static void init(Path cacheDir) throws IOException {
+  public static void init(Path cacheDir, int maxEntries, int maxBloatFactor) throws IOException {
```

Comment:
is it okay to change the signature of this public function? Maybe make another `init` with the old signature and default values if its possible that someone uses this